website/server/models/user/schema.js

```diff
@@ -357,6 +358,7 @@ let schema = new Schema({
     party: {type: Schema.Types.Mixed, default: () => {
       return {};
     }},
+    parties: {type: [{}], default: () => []},
```

Comment:
This should be an array of objects with specific values for the properties instead.